api/SVGTextContentElement.json

```diff
@@ -427,10 +427,10 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
This seems wrong. http://mdn-bcd-collector.appspot.com/tests/api/SVGTextContentElement doesn't test any instances and I think this is just a matter of the interface object not being exposed until Chrome 43, or something like that. The IDL file for this goes back to 2005: https://trac.webkit.org/changeset/10577/webkit

I'll revert the changes to this file to get this PR landed.